AFQ/api/group.py

```diff
@@ -187,7 +187,9 @@ def __init__(self,
             "Name": bids_description["Name"],
             "BIDSVersion": bids_description["BIDSVersion"],
             "PipelineDescription": {"Name": "pyAFQ",
-                                    "Version": pyafq_version}}
+                                    "Version": pyafq_version},
+            "GeneratedBy": [{"Name": op.basename(self.afq_path),
+                             "Version": pyafq_version}]}
```

Comment:
Also, while we're here, could we add something like: 
```suggestion
                                    "Version": pyafq_version},
            "GeneratedBy": [{"Name": op.basename(self.afq_path),
                             "Version": pyafq_version}], 
            "HowToAcknowledge": "Please cite this paper: https://apertureneuro.org/article/77465-evaluating-the-reliability-of-human-brain-white-matter-tractometry"}
```